src/internal/m365/collection/drive/url_cache.go

```diff
@@ -176,7 +176,7 @@ func (uc *urlCache) refreshCache(
 		}
 	}
 
-	du, err := pager.Results()
+	_, err := pager.Results()
```

Comment:
Can we update `TestURLCacheBasic` to also validate uc.prevDelta didn't change after the refresh?

Reply:
yea, was just thinking of adding a comment about that

Reply:
Thank you! I'll add this.